### PR TITLE
Done

### PR DESCRIPTION
**Pull request by MiwaGav**

### CODE REVIEW

File: homeworks/Mykhaylo.Gavrylyuk_MiwaGav/Homework_3/Funkciyi.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+function kvadrat(x, y, z) {
+    console.log(`1) ${x}^2 = ${x * x}`);
+    console.log(`2) ${y}^2 = ${y * y}`);
+    console.log(`3) ${z}^2 = ${z * z}`);
+}
+kvadrat(5, 15, 53);
```
Comment (markoffden) [inline homeworks/Mykhaylo.Gavrylyuk_MiwaGav/Homework_3/Funkciyi.js]: pls automate this with function

File: homeworks/Mykhaylo.Gavrylyuk_MiwaGav/Homework_3/Funkciyi.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    function massum(massiv) {
+        let sum = 0;
+        for (let j = 0; j < massiv.length; j++) {
+            sum += massiv[j];
+        }
+        return (sum);
+    }
+    console.log(massum(mas2));
+}
+summas(32, 4);
```
Comment (markoffden) [inline homeworks/Mykhaylo.Gavrylyuk_MiwaGav/Homework_3/Funkciyi.js]: ok, but could be optimized with `map()` and `reduce()` methods

**Comment by markoffden** [comment]
remove all duplicated files from your pull request

**Comment by MiwaGav** [comment]
> remove all duplicated files from your pull request
Підкажи будь-ласка як мені видалити зайві файли з пулреквесту

File: homeworks/Mykhaylo.Gavrylyuk_MiwaGav/Homework_3/Funkciyi.js (hunk 1 above)
Comment (MiwaGav) [reply homeworks/Mykhaylo.Gavrylyuk_MiwaGav/Homework_3/Funkciyi.js]: Зроблено

**Comment by markoffden** [comment]
Please, remove extra homeworks folder (homeworks/mykhaylo.gavrylyuk_miwagav)

**Pull request closed without merging**